package.json

```diff
@@ -72,11 +72,15 @@
     "typescript": "4.6.4"
   },
   "peerDependencies": {
-    "puppeteer": ">=16.2.0"
+    "puppeteer": ">=16.2.0",
+    "lighthouse": ">=10.0.0"
   },
   "peerDependenciesMeta": {
     "puppeteer": {
       "optional": true
+    },
+    "lighthouse": {
```

Comment:
since it's an optional peer dependency we should avoid any static non-type imports